mep3_simulation/webots_data/controllers/opponent_robot/opponent_robot.py

```diff
@@ -102,8 +120,19 @@ def main():
                 delta_x = math.cos(target_angle) * velocity_factor
                 delta_y = math.sin(target_angle) * velocity_factor
 
-                current_position[0] += delta_x
-                current_position[1] += delta_y
+                if not are_colliding(box_big_translation,
+                                     memristor_robot_translation) and supervisor.getName(
```

Comment:
Ova linija je čudno formatirana na kraju 🥴, savetujem da `and supervisor.getName() == 'opponent_box_big'` celo bude u sledećem redu.